box.go

```diff
@@ -69,13 +70,13 @@ func findBox(name string, order []LocateMethod) (*Box, error) {
 			return b, nil
 		case LocateWorkingDirectory:
 			// resolve absolute directory path
-			err := b.resolveAbsolutePathFromWorkingDirectory()
-			if err != nil {
+			if err = b.resolveAbsolutePathFromWorkingDirectory(); err != nil {
```

Comment:
I'd prefer to keep the assignment statement separate (on a line **before** the if statement)